docs/docs/reference/index.mdx

```diff
@@ -0,0 +1,296 @@
+</table>
+
+## [Testing](#testing)
+
+We're working to develop a standard around code coverage and testing frameworks. Look for this to be a required component, verified by tooling in the future.
+
+## [Security Requirements](#security)
+
+* A Pull Request must pass an automated security scan of the code and referenced libraries to be accepted.
+
+* The following practices are explicitly disallowed:
+
+    * The inclusion of additional SCRIPT tags
+
+    * Use of the *eval* command, the use of Function as a constructor (i.e. new Function(...))
+
+    * Be extremely careful when using dangerouslySetInnerHTML in React. In general, avoid it if you don’t have a solid reason for using it.
+
+* Code that appears to behave dangerously or in a manner different than documented in the README should be escalated to the Security team during code review. Examples include:
+
+    * Writing an unspecified object of data to NerdStorage
+
+    * Interacting with an outside URL w/o clear documentation around what is being retrieved
+
+    * Writing data to an outside source
+
+## [Legal Requirements](#legal)
+
+* The Nerdpack and it’s dependencies must but licensed under a permissive license such as Apache, MIT, BSD or similar license.
+
+* No code that leverages [viral license](https://en.wikipedia.org/wiki/Viral_license) projects will be accepted for inclusion in the Catalog. Only use permissively licensed open source packages unless given explicit exception.
+
+## [Design Requirements](#design)
+
+The following practices are explicitly disallowed
+
+* You MUST NOT override / amend core NR1 styles
+
+## [Design Guidelines](#guidelines)
+
+* Nerdpack developers SHOULD attempt to make use of the [NR1 visual components](https://developer.newrelic.com/client-side-sdk/index.html#components/BlockText) whenever possible. Usage of third party visual libraries is allowed, but they may not be able to take advantage of ongoing platform improvements and features.
+
+## [Package Naming](#naming)
+
+* Projects SHOULD NOT include the word *New Relic *or *newrelic* in package name or title.
+
+* Projects SHOULD be named with human-readable, functionally descriptive names.
+
+* Project names MUST be unique within the catalog to support the internal project mapping of submodules to uuid’s.
+
+* Project names SHOULD use title case (e.g. "Deployment Analyzer", not “Deployment analyzer”).
+
+* Profanity or crude language is disallowed.
+
+## [Documentation Guidelines](#documentation)
+
+* Nerdpack developers MUST provide the following metadata for each version of their application
+
+    * tagline for the application (at most 30 characters)
+
+    * source repository URL (at most 1,000 characters)
+
+    * application details (at most 1,000 characters)
+
+        * Markdown allowed, no images or videos
+
+    * semantic Version number (e.g. 1.1.0)
+
+    * description of what’s new in the current version (at most 500 characters)
+
+        * Markdown allowed, no images or videos
```

Comment:
Same as above - If this is describing the config.json, I think this is incorrect since that's all plaintext. The only thing I recall being supported was \n for line breaks.